components/validator/cmd/main.go

```diff
@@ -28,6 +29,43 @@ func main() {
 	app.Description = "Service for generating and submitting L2 output checkpoints to the L2OutputOracle contract as an L2 Output Submitter, " + "detecting and correcting invalid L2 outputs as a Challenger to ensure the integrity of the L2 state."
 
 	app.Action = curryMain(Version)
+	app.Commands = []cli.Command{
+		{
+			Name: "balance",
```

Comment:
How about removing the `balance` command and having `deposit`, `withdraw`, and `unbond` come here?

Reply:
like this.

`kroma-validator deposit <amount>`
`kroma-validator withdraw <amount>`

Reply:
I made the changes you mentioned, but I didn't move all the functions in `balance/cmd.go` file to `validator/main.go` file to consider the possibility of other categories of commands being added in the future.

Reply:
Good. This is what i expected.